src/test/java/com/razorpay/CustomerClientTest.java

```diff
@@ -295,4 +296,168 @@ public void testDeleteToken() throws IOException, RazorpayException {
             assertTrue(false);
         }
     }
+
+    /**
+     * Add Bank Account
+     */
+    @Test
+    public void testaddBankAccount() throws RazorpayException{
+
+        JSONObject request = new JSONObject("{\n" +
+                "    \"ifsc_code\" : \"UTIB0000194\",\n" +
+                "    \"account_number\"         :\"916010082985661\",\n" +
+                "    \"beneficiary_name\"      : \"Pratheek\",\n" +
+                "    \"beneficiary_address1\"  : \"address 1\",\n" +
+                "    \"beneficiary_address2\"  : \"address 2\",\n" +
+                "    \"beneficiary_address3\"  : \"address 3\",\n" +
+                "    \"beneficiary_address4\"  : \"address 4\",\n" +
+                "    \"beneficiary_email\"     : \"random@email.com\",\n" +
+                "    \"beneficiary_mobile\"    : \"8762489310\",\n" +
+                "    \"beneficiary_city\"      :\"Bangalore\",\n" +
+                "    \"beneficiary_state\"     : \"KA\",\n" +
+                "    \"beneficiary_country\"   : \"IN\"\n" +
+                "}");
+
+        String mockedResponseJson = "{\n" +
+                "    \"id\": \"ba_LSZht1Cm7xFTwF\",\n" +
+                "    \"entity\": \"bank_account\",\n" +
+                "    \"ifsc\": \"ICIC0001207\",\n" +
+                "    \"bank_name\": \"ICICI Bank\",\n" +
+                "    \"name\": \"Gaurav Kumar\",\n" +
+                "    \"notes\": [],\n" +
+                "    \"account_number\": \"XXXXXXXXXXXXXXX0434\"\n" +
+                "}";
```

Comment:
same here, is there any chance that you can create mockResponse using any class/POJO, and when passing to mockResponseFromExternalClient(), pass the object as string using toString() method?

Reply:
added POJO to JSONObject

Reply:
+1